### PR TITLE
HIPP-1755: Add fetch credentials logic on the environment page

### PR DESCRIPTION
**Pull request by victorarbuesmallada**

### CODE REVIEW

**Comment by victorarbuesmallada** [comment]
test this please

File: app/views/application/EnvironmentsView.scala.html [hunk 1]
```diff
@@ -42,26 +52,36 @@ <h2 class="govuk-heading-m">Credentials</h2>
...
-                <p class="govuk-body govuk-!-margin-bottom-3">
-                    <strong>@messages("environments.clientSecret"):</strong>
-                    @credential.clientSecret.getOrElse("")
-                </p>
-                <p class="govuk-body govuk-!-margin-bottom-3">
-                    <strong>@messages("environments.created"):</strong>
-                    <span class="utcDateTime">@ViewUtils.formatLocalDateTimeContainingUtc(credential.created)</span>
-                </p>
-            </div>
+        @if(hipEnvironment.isProductionLike && maybeCredentials.isEmpty) {
```
Comment (victorarbuesmallada) [inline app/views/application/EnvironmentsView.scala.html:62]: not sure if I need this condition here
`hipEnvironment.isProductionLike`

**Review by PaulCDurham — CHANGES_REQUESTED** [review]
The main thing is to always show credentials. If there is an error fetching them then show the obfuscated secret fragment.

**Review by llmikeyj — APPROVED** [review]

**Comment by victorarbuesmallada** [comment]
test this please

**Review by PaulCDurham — APPROVED** [review]

**Pull request merged**